velox/experimental/cudf/tests/TableScanTest.cpp

```diff
@@ -439,3 +481,54 @@ TEST_F(TableScanTest, splitOffset) {
           .assertEmptyResults(),
       "CudfHiveDataSource cannot process splits with non-zero offset");
 }
+
+TEST_F(TableScanTest, s3DataSource) {
```

Comment:
We should add a new file `S3TableScanTest.cpp` that uses the S3Test as the base class.
Since this test depends on s3fs, we need to set the correct dependencies.
I think we will be adding a lot more tests for S3 in the future.

Reply:
https://github.com/facebookincubator/velox/pull/15391